GitUpKit/Views/GIMapViewController+Operations.m

```diff
@@ -225,7 +225,9 @@ - (void)swapCommitWithChild:(GCHistoryCommit*)commit {
 
 - (void)squashCommitWithParent:(GCHistoryCommit*)commit {
   if ([self checkCleanRepositoryForOperationOnCommit:commit]) {
-    [self _promptForCommitMessage:_CleanedUpCommitMessage(commit.message)
+    GCHistoryCommit* parentCommit = commit.parents.firstObject;
```

Comment:
In the GitUp UI, the "Squash With Parent" action is disabled when a commit does not have a parent. Outside of GitUp usage this of course result in an error but I believe there are other places that assume `commit.parents` will be non-null.